sauce/features/accounts/right-click-to-edit/index.js

```diff
@@ -0,0 +1,80 @@
+import { Feature } from 'toolkit/core/feature';
+import { getCurrentRouteName } from 'toolkit/helpers/toolkit';
+
+export class RightClickToEdit extends Feature {
+
+  isCurrentlyRunning = false;
+
+  injectCSS() {
+    return require('./index.css');
+  }
+
+  shouldInvoke() {
+    return getCurrentRouteName().indexOf('account') !== -1;
+  }
+
+  // Supporting functions,
+  // or variables, etc
+  displayContextMenu(event) {
+    let $element = $(this);
+    // check for a right click on a split transaction
+    if ($element.hasClass('ynab-grid-body-sub')) {
+      // select parent transaction
+      $element = $element.prevAll('.ynab-grid-body-parent:first');
+    }
+
+    if (!$element.hasClass('is-checked')) {
+      // clear existing, then check current
+      $('.ynab-checkbox-button.is-checked').click();
+      $element.find('.ynab-checkbox-button').click();
+    }
+
+    // make context menu appear
+    $('.accounts-toolbar-edit-transaction').click();
+
+    // determine if modal needs to be positioned above or below clicked element
+    let height = $('.modal-account-edit-transaction-list .modal').outerHeight();
+    let below = (event.pageY + height) > $(window).height() ? false : true; // eslint-disable-line no-unneeded-ternary
```

Comment:
Unless I'm missing something, the linter's complaining because this line could be:

```javascript
const below = (event.pageY + height) <= $(window).height();
```

Which reads quite a bit better to me.